BootloaderCommonPkg/Library/LinuxLib/LinuxLib.c

```diff
@@ -314,6 +318,15 @@ UpdateLinuxBootParams (
     E820Entry++;
   }
   Bp->E820Entries = (UINT8)MapInfo->Count;
+
+  GuidHob = GetFirstGuidHob (&gLoaderSystemTableInfoGuid);
+  if (GuidHob != NULL) {
+    SystemTableInfo = (SYSTEM_TABLE_INFO *)GET_GUID_HOB_DATA (GuidHob);
+
+    AsciiSPrint (ParamValue, sizeof (ParamValue), " acpi_rsdp=0x%lx", SystemTableInfo->AcpiTableBase);

```

Comment:
Will this prevent user from overriding acpi_rsdp in command line in case they want ?

Reply:
You are right, this will prevent user option since kernel command parameters are parsed sequentially. To avoid this, let me add acpi_rsdp only if the kernel command line does not have it.

Reply:
Updated to check "acpi_rsdp=" existence before appending it.